site/src/sphinx/advanced-logging.rst

```diff
@@ -134,3 +139,25 @@ as the 3rd component of the ``<export />`` element in the XML configuration:
       </appender>
       ...
     </configuration>
+
+Customizing MDC keys
+--------------------
+You can override the pre-defined the MDC key by prepending an alias and a equals sign (=) to it.
+For example, if you want to change ``req.id`` to ``request_id``, use ``request_id=req.id``.
+
+.. code-block:: xml
+
+    <?xml version="1.0" encoding="UTF-8"?>
+    <configuration>
+      ...
+      <appender name="RCEA" class="com.linecorp.armeria.common.logback.RequestContextExportingAppender">
+        ...
+        <export>remote_id=remote.id</export>
+        <export>UA=req.http_headers.user-agent</export>
+        <export>some_value=attr:com.example.AttrKeys#SOME_VALUE</exports>
+        ...
+      </appender>
+      ...
+    </configuration>
+
+Note that this cannot be used with wildcard expressions ``*`` or ``req. *``.
```

Comment:
How about:

    Note that a custom MDC key cannot be used with a wildcard expression such as * or req.*

.. because `this` is somewhat vague.

I also see a space between `req.` and `*`.